app.php

```diff
@@ -64,148 +63,182 @@ function createCollection()
             ]
         ]
     );
+
     $collectionId = $response['id'];
```

Comment:
convert it to string so it can run like
`$collectionId = (string)$response['id']`
createDoc() function takes string as argument.